app/views/spree/admin/return_authorizations/new.html.haml

```diff
@@ -1,4 +1,5 @@
 = render partial: 'spree/admin/shared/order_tabs', locals: { current: 'Return Authorizations' }
+= render partial: 'spree/admin/shared/order_page_title', locals: { current: 'Return Authorizations' }
```

Comment:
Hey Robin, I think this is still a previous version of the code. We already discussed this and you fixed it, the call to order_page_title doesnt need the current variable.

Reply:
done